db/migrations/postgres/000024_create_token_tables.up.sql

```diff
@@ -0,0 +1,14 @@
+BEGIN;
+CREATE TABLE tokenpool (
+  seq            SERIAL          PRIMARY KEY,
+  id             UUID            NOT NULL,
+  namespace      VARCHAR(64)     NOT NULL,
+  name           VARCHAR(64)     NOT NULL,
+  pool_id        VARCHAR(80)     NOT NULL,
+  is_fungible    SMALLINT        DEFAULT 1
```

Comment:
Couldn't decide whether to store this as a boolean or a string enum. We deal with strings in the API (and theoretically could support more types than just "fungible" and "nonfungible"), but this seemed simpler? Not sure if we have a precedent.

Reply:
@peterbroadhurst says it should be a 64 char string